src/Service/Users.php

```diff
@@ -49,7 +49,8 @@ class Users {
   public function __construct(ConfigFactory $configFactory,
                               OpenamApiClient $openamApiClient,
                               EventDispatcherInterface $eventDispatcher) {
-    $this->config = $configFactory->get('openam_api_operations.settings');
+    $openamConfig = $configFactory->get('openam_api.settings');
+    $this->config = $openamConfig->get('openam_api_operations');
```

Comment:
this is defined as 'openam_api_operations.settings' in the schema! Are you seeing any issues with the mismatched naming convention?

Reply:
No I dint see any issues. But I will change it.

Reply:
Updated.

Reply:
Please can you rename `$this->config` to something a little more descriptive? Its not the entire "config", its only operations, therefore a more logical and descriptive name would be:
`$this->openamApiOperations`

Reply:
Updated.